packages/client-core/src/admin/components/Party/CreateParty.tsx

```diff
@@ -1,32 +1,37 @@
 import React, { useState, useEffect } from 'react'
-import Fade from '@mui/material/Fade'
-import Modal from '@mui/material/Modal'
-import styles from '../Admin.module.scss'
-import classNames from 'classnames'
-import Autocomplete from '@mui/material/Autocomplete'
-import TextField from '@mui/material/TextField'
 import { LocationService } from '../../services/LocationService'
 import { useDispatch } from '../../../store'
 import { useAuthState } from '../../../user/services/AuthService'
 import { PartyService } from '../../services/PartyService'
 import { InstanceService } from '../../services/InstanceService'
 import DialogContentText from '@mui/material/DialogContentText'
-import DialogActions from '@mui/material/DialogActions'
-import Button from '@mui/material/Button'
-import Typography from '@mui/material/Typography'
-import { PartyProps } from './variables'
-import { usePartyStyle } from './style'
+import { PartyProps } from '../../common/variables/party'
+import { useStyles } from '../../styles/ui'
 import { useLocationState } from '../../services/LocationService'
 import { useInstanceState } from '../../services/InstanceService'
 import { Instance } from '@xrengine/common/src/interfaces/Instance'
+import CreateModel from '../../common/CreateModel'
+import Paper from '@mui/material/Paper'
+import FormControl from '@mui/material/FormControl'
+import MenuItem from '@mui/material/MenuItem'
+import Select from '@mui/material/Select'
+import { validateForm } from '../../common/Validation/formValidation'
+import { useFetchAdminInstance } from '../../common/hooks/Instance.hooks'
+import { useFetchAdminLocations } from '../../common/hooks/Location.hooks'
 
 const CreateParty = (props: PartyProps) => {
-  const classes = usePartyStyle()
+  const classes = useStyles()
   CreateParty
   const { open, handleClose } = props
 
-  const [location, setLocation] = useState('')
-  const [instance, setInstance] = React.useState('')
+  const [state, setState] = useState({
```

Comment:
can we please use some other name `newParty` or something else. `state` just seems like a keyword and creates confusion.